pkg/resource/prunercronjob.go

```diff
@@ -116,6 +124,7 @@ func (gcj *generatorPrunerCronJob) expected() (runtime.Object, error) {
 										"--certificate-authority=/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt",
 										fmt.Sprintf("--keep-tag-revisions=%d", gcj.getKeepTagRevisions(cr)),
 										fmt.Sprintf("--keep-younger-than=%s", gcj.getKeepYoungerThan(cr)),
+										fmt.Sprintf("--prune-registry %t", gcj.getPruneRegistry(rcr)),
```

Comment:
`--prune-registry=%t`, to be consistent.

Reply:
updated in next push

Reply:
@coreydaley since 4.4 is not fast forwarding from master, please create a separate PR for the `--prune-registry` flag fix, along with a BZ if QE hasn't created one already.

We will want to fix this behavior before code freeze if possible.